srcpkgs/WildMidi/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'WildMidi'
+pkgname=WildMidi
+version=0.4.2
+revision=1
+wrksrc="wildmidi-wildmidi-${version}"
+build_style=cmake
+makedepends="alsa-lib-devel libopenal-devel"
+short_desc="Core softsynth library that can be used with other applications"
+maintainer="oreo639 <oreo6391@gmail.com>"
+license="GPL-3.0-or-later, LGPL-3.0-only"
+homepage="https://www.mindwerks.net/projects/wildmidi/"
+distfiles="https://github.com/Mindwerks/wildmidi/archive/wildmidi-${version}.tar.gz"
+checksum=551d43cb6de6019885f933a20b6f3205a92814f50da8b0d8bceac002b9a8109d
+
+WildMidi-devel_package() {
+	short_desc+=" - development files"
+	depends="${makedepends} WildMidi>=${version}_${revision}"
+	pkg_install() {
```

Comment:
is missing

```
vmove usr/include
vmove usr/share/man/man3
vmove "usr/lib/*.so"
```

Reply:
The midi player requires the wildmidi library. But yeah, I should have done it for the header file and man3. I'll get on that asap.

Reply:
the *.so will match only 

`/usr/lib/libWildMidi.so -> /usr/lib/libWildMidi.so.2`

which is a symlink available on WildMidi which remais with

```
/usr/lib/libWildMidi.so.2.0.1 # Library proper
/usr/lib/libWildMidi.so.2 -> /usr/lib/libWildMidi.so.2.0.1 # Symlink which packages will link against and it is recorded on common/shlibs
```

Reply:
I was doing something wrong. Thank you.